frontend/app/configuration/Config.scala

```diff
@@ -49,14 +49,9 @@ object Config {
   def idWebAppSigninUrl(uri: String): String =
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
 
-  def idWebAppRegisterUrl(uri: String, abTestVariant: CheckoutFlowVariant = A): String ={
-
-    val idMember = "clientId" -> abTestVariant.identitySkin
-
-    (idWebAppUrl / "register") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
-  }
-
-
+  def idWebAppRegisterUrl(uri: String, abTestVariant: CheckoutFlowVariant = A): String =
+    (idWebAppUrl / "register") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & "clientId" -> abTestVariant.identitySkin
```

Comment:
@rtyley I tested in dev and the evaluation was correct. Just in case (and to improve the 	
legibility of the code), I added a pair of parentheses.